apps/web/lib/ErrorBoundary.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ErrorInfo } from "react";
+
+class ErrorBoundary extends React.Component<
```

Comment:
ErrorBoundary IMO is a good idea, but it should be updated to use react hooks :) Not sure if you want to do that now or in a future iteration

Reply:
AFAIK it HAS to be a class since there's no `componentDidCatch` equivalent in hook.

Reply:
I'll save you the time 😜
https://stackoverflow.com/a/59932120/6297100